roles/artemis/defaults/main.yml

```diff
@@ -245,6 +245,9 @@ hazelcast_address: "{% if is_multinode_install is defined and is_multinode_insta
 artemis_eureka_urls: "{% if registry.url is defined and registry.url is not none %}http://admin:${jhipster.registry.password}@{{ registry.url }}:8761/eureka/{% endif %}"
 artemis_eureka_instance_id: "{{ node_id }}"
 
+node_short_name: unnamed-artemis-node
+node_display_name: Unnamed Artemis Node
```

Comment:
You might also move the parameter to `continuous_integration.localci`.

Reply:
No this is correct